release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -434,26 +435,36 @@ The Docs Lead is responsible for updating the Kubernetes API reference, kubectl,
 If you need help in building the reference documentation, reach out on Slack #sig-docs.
 
 ### Generate the reference documentation
+Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)).
 
-Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)). Before running the script, modify `reference.yml` to checkout the Kubernetes future release branch.
-
-![Update kubernetes release branch](pics/reference-yaml.png)
-
-The first step is to famialiarize yourself with the `website/updated-imported-docs/update-imported-docs.py` script
+The first step is to familiarize yourself with the `website/updated-imported-docs/update-imported-docs.py` script
 and the instructions for ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/quickstart/)).
 
-The reference documentation build depends upon a valid release tag.
+The reference documentation build depends upon a valid [release tag](https://github.com/kubernetes/kubernetes/tags).
 When a Kubernetes release candidate version tag such as `v1.19-rc.2` is created, you can consider building
 the reference documentation. Next:
 
 - Create a branch for this work based off of the `dev-[future release]` branch.
+```
+git checkout dev-[future-release]
+git fetch upstream dev-[future-release]
+git rebase upstream dev-[future-release]
+git checkbout -b dev-[future-release]-ref-doc
+```
 - Run the `update-imported-docs.py` script providing the release tag to build the reference documentation.
-- Commit the generated files.
+```
+python website/update-imported-docs/update-imported-docs.py reference.yml [release tag]
+```
 
-![Copy and rename screenshot](pics/copy-rename-dir.png)
+- Commit the generated files.
+```
+git add .
+git commit -m "Generate reference doc for [release tag]"
+git push --set-upstream origin dev-[future-release]-ref-doc
+```
 
 You can expect to maintain this branch with periodic updates to the reference documentation using the latest `rc` tag.
-Merge your branch to the `dev-[future release]` branch just before the final release.
+Merge your branch `dev-[future-release]-ref-doc` to the `dev-[future release]` branch just before the final release.
 When the release is complete, you should build the reference documentation again using the final release tag and merge to the `master` branch.
```

Comment:
Line 468, I think this is optional? The content may not change, but the build can be verified with the actual release tag (not release candidate). Every time you regenerate, the API reference will generate a new timestamp.

Reply:
How does this sound?
```
If the content has changed since the last build of the reference documentation, when the release is complete, you should
build the reference documentation again using the final release tag and merge to the `master` branch.
```